test/test_bazel.py

```diff
@@ -0,0 +1,15 @@
+# Owner(s): ["module: bazel"]
+
+"""
+This test module contains a minimalistic "smoke tests" for the bazel build.
+
+Currently it doesn't use any testing framework (i.e. pytest)
```

Comment:
is this because rules.requirement() is a no-op and we can't insert a dependency on pytest? I'd like to use a newer version of rules_python and add dependencies to Bazel. Is that something that will be compatible with how Cruise is using PyTorch?